lib/inputs/binaries/static/index.ts

```diff
@@ -1,23 +1,36 @@
 import { ExtractAction, ExtractedLayers } from "../../../extractor/types";
 import { streamToHash } from "../../../stream-utils";
 
+export const getOpenJDKBinariesFileContentAction: ExtractAction = {
+  actionName: "java",
+  fileNamePattern: "**/java",
```

Comment:
there's several different ```java``` binaries often, for example in ```.../bin/java``` but also ```.../bin/jre/java```.
do we need/want to collect both? worth checking

Reply:
as per chat, we're collecting all the java binaries